src/helper/stepsDefinitions.ts

```diff
@@ -130,7 +132,7 @@ function getBreathSteps(storeKey: string, country: string, patientId?: string) {
             if (patientId) {
               return `${baseUrl}/thank-you`;
             }
-            return `${baseUrl}/questionary/step1a`;
+            return country === 'Japan' ? `${baseUrl}/step-record/speech` : `${baseUrl}/questionary/step1a`;
```

Comment:
As you included `Japan` in `allowSpeechIn`, this condition ` if (!allowSpeechIn.includes(country)) {` will be false always as allowSpeechIn includes Japan. You don't need this check on the return.